src/bot/commands/misc/avatar.js

```diff
@@ -1,24 +1,25 @@
-// Shows the avatar of the user provided
+// Shows the server avatar of the user provided
 
 import {Command} from 'yuuko';
-import {parseUser} from '../../util/discord';
+import {parseGuildMember} from '../../util/discord';
 
 const command = new Command('avatar', async (msg, args) => {
 	const MAX_AVATAR_IMAGE_SIZE = 512;
 
 	if (args.length === 0) {
-		msg.channel.createMessage(msg.author.dynamicAvatarURL('', MAX_AVATAR_IMAGE_SIZE)).catch(() => {});
+		msg.channel.createMessage(`${msg.member.avatarURL.split('?size=')[0]}?size=${MAX_AVATAR_IMAGE_SIZE}`).catch(() => {});
 		return;
 	}
-	const [member] = await parseUser(args.join(' '), msg.channel.guild, msg.author);
+	const [member] = await parseGuildMember(args[0], msg.channel.guild, msg.member);
```

Comment:
> What's the rationale for passing only the first argument here instead of all arguments?

This command only accepts at most one argument so I didn't think there was a need to join the arguments. Is there any case where this approach won't work?

Reply:
The parse functions can interpret `user#discrim` references that aren't real mentions, and usernames can have spaces in them. `args` is naive and just splits a string on all spaces; it's mostly useless if a single logical "argument" (a member, for example) can be represented in text by a string that contains spaces. That's why it's always stitched back together with `.join(' ')` before being passed anywhere.

Reply:
Got it! I'll change it back!

Reply:
(resolved in bc4046c)